src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -81,6 +81,23 @@ const en = ({ isDivorce, userCase, partner }: CommonContent) => ({
     )} after your application has been checked.
     This will have the time, date and court your conditional order will be pronounced.`,
   },
+  conditionalOrderRejected: {
+    line1: `The court is not yet satisfied you are entitled to ${
+      isDivorce ? 'get divorced' : 'end your civil partnership'
+    }. You need to read the court’s feedback and update your application, before you can continue.`,
+    line2: 'The court’s feedback',
+    line3: `${userCase.coRefusalRejectionAdditionalInfo}`,
+    part1: 'You can download a copy of the court’s full',
+    part2: 'Refusal Order PDF.',
+    downloadReference: 'Refusal-Order',
+    link: '/downloads/conditional-order-refusal',
+    line4: 'What you need to do',
+    line5: 'You will need to change the application, and submit it to the court again.',
+    line6:
+      'You will receive a paper copy of the application in the post. It will include a letter with details of how to update the application and send it back to the court.',
+    line7: `You will need to agree the changes with your ${partner} before sending it back to the court.`,
+    line8: 'You will also need to pay a £95 amendment fee',
```

Comment:
better to not hardcode the £95 if we can avoid it. We put other fees down in config/default.yaml but you'll have to find the correct name for this fee

Reply:
@lara-mpt I used the following updateApplication as the fee. Its returning £95, which is the correct one I believe.